apps/prairielearn/elements/pl-string-input/pl-string-input.mustache

```diff
@@ -39,8 +39,8 @@
                 autocomplete="off"
                 autocorrect="off"
                 {{^editable}}disabled{{/editable}}
-                aria-describedby="pl-string-input-{{uuid}}-suffix"
```

Comment:
Nothing for the label here?

Reply:
This one uses `label=for` correctly too. Is this a pattern we want to switch to, using `label` instead of `span`?

Reply:
Added a fallback in f456869a190d52c527c4c169a14235ccbf00d45b